index.html

```diff
@@ -35,9 +35,290 @@ <h3>I am a software developer</h3>
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
+                <p class="primary">About Me</p>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=""></section>
+                <section class="button">
+                    <p class="get">Get My Resume</p>
+
+                </section>
+                <section class="divider">
+                    <div class="divi"></div>
+                </section>
+                <section class="skill">
+                    <ul >
+                        <li class="li-items">
+                            <div class="li-main">
+                                <div class="tri"></div>
+                                <p>languages</p>
+
+                            </div>
+                            <div>
+                                <ul class="lan">
+                                    <div class="mjava"><li class="java">Javascript</li></div>
+                                    <div class="mruby"><li class="ruby">Ruby</li></div>
+                                    <div class="mhtml"><li class="html">html</li></div>
+                                    <div class="mcss"><li class="css">css</li></div>
+                                </ul>
+                            </div>
+                        </li>
+                        <li class="li-items1">
+                            <div class="li-main">
+                                <div class="squ"></div>
+                                <p>Frameworks</p>
+
+                            </div>
+                            <div>
+                                <ul class="fra">
```

Comment:
- [ ] Please align the list of technologies to look like the Figma template. You are expected to stick to the design template as much as possible.
### Your design
![4](https://user-images.githubusercontent.com/94551484/192065483-776d8213-fa12-49a0-98ef-9e198746781b.png)
### Figma template
![f4](https://user-images.githubusercontent.com/94551484/192065491-d4ab0c3c-1f2b-4d15-a177-8f16be6042a8.png)